simul/build.go

```diff
@@ -116,25 +117,19 @@ func RunTests(deployP platform.Platform, name string, runconfigs []*platform.Run
 	}
 
 	mkTestDir()
-	var f *os.File
 	args := os.O_CREATE | os.O_RDWR | os.O_TRUNC
 	// If a range is given, we only append
 	if simRange != "" {
 		args = os.O_CREATE | os.O_RDWR | os.O_APPEND
 	}
-	f, err := os.OpenFile(testFile(name), args, 0660)
-	if err != nil {
-		log.Fatal("error opening test file:", err)
-	}
+	files := []*os.File{}
 	defer func() {
-		if err := f.Close(); err != nil {
-			log.Error("Couln't close", f.Name())
+		for _, f := range files {
+			if err := f.Close(); err != nil {
+				log.Error("Couln't close", f.Name())
```

Comment:
I don't know if that applies here, but error strings should not be capitalized

Reply:
Usually the logs are written as a correct English sentence. You're right about the errors though.